package/system/procd/patches/0012-use-os-release.patch

```diff
@@ -0,0 +1,57 @@
+From 51c2d32c98b2ae92d00820c3734dec525766b1fa Mon Sep 17 00:00:00 2001
+From: Florian Eckert <fe@dev.tdt.de>
+Date: Tue, 18 Jun 2024 15:05:45 +0200
+Subject: [PATCH 1/2] system: replace openwrt_release with os-release
```

Comment:
FYI there is https://github.com/openwrt/procd where this belongs.

Reply:
Is procd now maintained on github?
I was not aware of that.
I will open a PR asap to procd
Thanks :+1:

Reply:
If the development is now on github, then we should remove in the repository description on github
the **MIRROR** `-->[MIRROR]<-- OpenWrt service / process manager`.

Reply:
> If the development is now on github, then we should remove in the repository description on github the **MIRROR** `-->[MIRROR]<-- OpenWrt service / process manager`.

It works the same way as this repository. You can create PRs, but they won't get merged via github and instead via the repo on git.openwrt.org.

Reply:
What @ynezz mainly wanted to express is that you shouldn't add patches for OpenWrt projects in OpenWrt itself, and instead submit the changes directly to the appropriate sub project.

Whether this happens via a PR on github or via patches to the mailing list is secondary.